Makefile

```diff
@@ -39,3 +39,23 @@ check-examples:
 		done; \
 	done
 
+.PHONY: update-crds
+update-crds:
```

Comment:
Let's call them `update-operator-crds` and `check-operator-crds`?

Reply:
Done